test/integration/galera_configuration/controls/galera_spec.rb

```diff
@@ -0,0 +1,49 @@
+include_dir = '/etc/mysql/conf.d'
+include_dir = '/etc/my.cnf.d' if os.family == 'redhat'
```

Comment:
This isn't what I suggested originally. Can you please make the change I suggested instead?

Reply:
Having a `case` statement for one option feels excessive, doing it like this is cleaner and pretty common for a single override.

I agree if there was another override happening then a `case` statement makes sense.